pages/Category/components/Products/README.md

```diff
@@ -8,28 +8,10 @@ Depending on the view mode the list or grid is hidden and not rendered.
 
 ```
 import { Products } from 'Templates/components';
```

Comment:
The templates constant doesn't exist anymore

Reply:
Import corrected